oragono.yaml

```diff
@@ -188,6 +188,45 @@ server:
             # - "192.168.1.1"
             # - "2001:0db8::/32"
 
+    # IP cloaking hides users' IP addresses from other users and from channel admins
+    # (but not from server admins), while still allowing channel admins to ban
+    # offending IP addresses or networks. In place of hostnames derived from reverse
+    # DNS, users see fake domain names like pwbs2ui4377257x8.oragono. These names are
+    # generated deterministically from the underlying IP address, but if the underlying
+    # IP is not already known, it is infeasible to recover it from the cloaked name.
+    ip-cloaking:
+        # whether to enable IP cloaking
+        enabled: false
+
+        # fake TLD at the end of the hostname, e.g., pwbs2ui4377257x8.oragono
+        netname: "oragono"
+
+        # secret key to prevent dictionary attacks against cloaked IPs
+        # any high-entropy secret is valid for this purpose:
+        # you MUST generate a new one for your installation.
+        # suggestion: use the output of this command:
+        # python3 -c "import secrets; print(secrets.token_urlsafe())"
+        # note that rotating this key will invalidate all existing ban masks.
+        secret: "siaELnk6Kaeo65K3RCrwJjlWaZ-Bt3WuZ2L8MXLbNb4"
```

Comment:
(if we aren't already), we should make sure that if this default secret is being used that users receive a message on connection noting it (like the current message which states when logging of raw traffic is enabled on the server). While I'm not against Python at all, we should expose an Oragono command to generate these to make sure it's an all-in-one package no matter the OS or existing software installed.